ttnn/cpp/ttnn/operations/eltwise/binary/binary_pybind.hpp

```diff
@@ -431,11 +430,53 @@ void bind_binary_overload_operation(py::module& module, const binary_operation_t
             const std::optional<MemoryConfig>& memory_config) {
                 return self(input_tensor_a, input_tensor_b, memory_config); },
             py::arg("input_tensor_a"),
-            py::arg("inputr_tensor_b"),
+            py::arg("input_tensor_b"),
```

Comment:
use the py::arg names given here consistently everywhere in the R"doc. 
if you want to keep the name common for an arg that could be tensor or float, maybe use "input_b" instead of "other", since the first arg ends with  _a 
maybe input_a , input_b ?